multi_subject/manual_correction.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/env python
+#
+# Script to perform manual correction of segmentations and vertebral labeling.
```

Comment:
This doesn't really affect the SCT Course, but in the future it would be good to expand [this page](https://spinalcordtoolbox.com/user_section/tutorials/analysis-pipelines-with-sct/handling-issues.html) to include the new material you've written for the SCT Course. :blush: